torchaudio/functional/functional.py

```diff
@@ -1975,12 +1983,20 @@ def rtf_power(psd_s: Tensor, psd_n: Tensor, reference_channel: Union[int, Tensor
             If the dtype is ``Tensor``, the dimension is `(..., channel)`, where the ``channel`` dimension
             is one-hot.
         n_iter (int): number of iterations in power method. (Default: ``3``)
+        diagonal_loading (bool, optional): whether to apply diagonal loading to psd_n
+            (Default: ``True``)
+        diag_eps (float, optional): The coefficient multiplied to the identity matrix for diagonal loading
+            (Default: ``1e-7``)
+        eps (float, optional): a value to avoid the correlation matrix is all-zero (Default: ``1e-8``)
```

Comment:
can this docstring be more helpful? does `a value added to the denominator in the beamforming weight computation.` from #2368 make sense here, and is it worth adding that this is only used for the case when `diagonal_loading=True`?

Reply:
Makes sense. I will align the eps docstring in the functions and modules.

Reply:
The `eps` here is for diagonal loading, which is confusing with `eps` in computing beamforming weight. I decided to exclude it from the API and use the default value in `_tik_reg`.